tests/test_invalid_nixpkgs_package.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e
```

Comment:
```suggestion
#!/usr/bin/env bash
set -euo pipefail
```
This script uses bash-isms.